src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import core.basesyntax.io.FileReader;
+import core.basesyntax.io.FileReaderImpl;
+import core.basesyntax.io.FileWriter;
+import core.basesyntax.io.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.servise.DataConverter;
+import core.basesyntax.servise.DataConverterImpl;
+import core.basesyntax.servise.OperationStrategy;
+import core.basesyntax.servise.OperationStrategyImpl;
+import core.basesyntax.servise.ReportGenerator;
+import core.basesyntax.servise.ReportGeneratorImpl;
+import core.basesyntax.servise.ShopService;
+import core.basesyntax.servise.ShopServiceImpl;
+import core.basesyntax.servise.operation.BalanceOperation;
+import core.basesyntax.servise.operation.OperationHandler;
+import core.basesyntax.servise.operation.PurchaseOperation;
+import core.basesyntax.servise.operation.ReturnOperation;
+import core.basesyntax.servise.operation.SupplyOperation;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    private static final String DIRECTORY_PATH = "./src/main/java/core/basesyntax";
+    private static final String INPUT_FILE_PATH = DIRECTORY_PATH + "/transactions.csv";
+    private static final String OUTPUT_FILE_PATH = "finalReport.csv";
```

Comment:
Avoid using hardcoded values for file paths. Instead, you could pass these as arguments to your program or read them from a configuration file or environment variables. This would make your code more flexible and easier to run in different environments.